lib/fastlane/plugin/firebase_app_distribution/client/aab_info.rb

```diff
@@ -0,0 +1,42 @@
+class AabInfo
+  # AAB states
+  class AabState
+    UNSPECIFIED = "AAB_STATE_UNSPECIFIED"
+    PLAY_ACCOUNT_NOT_LINKED = "PLAY_ACCOUNT_NOT_LINKED"
+    NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT = "NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT"
+    APP_NOT_PUBLISHED = "APP_NOT_PUBLISHED"
+    PLAY_IAS_TERMS_NOT_ACCEPTED = "PLAY_IAS_TERMS_NOT_ACCEPTED"
+    ACTIVE = "ACTIVE"
+    UNAVAILABLE = "AAB_STATE_UNAVAILABLE"
```

Comment:
This looks like an enum that might be present in backend code, eg: http://google3/java/com/google/firebase/console/web/modules/appdistro/ng2/view/onboarding_state.ts?l=52-60&rcl=389870755 

It'd probably be good to crossref this fastlane definition from that location with a suggestion to keep these enums in sync.

Reply:
Agreed, but I didn't want to throw an internal google URL in this public code. We'll know to update this code if that enum is ever changed (we'd scrub through all of our client code in that case).

Reply:
I agree about not throwing an internal google URL in this public code.  To clarify my suggestion, it was to explicitly grab a link to _this_ code and drop it into the source file in codesearch.  I trust that you would scrub through the client code, but if I were making a change to the enum, I wouldn't know where to look.

Reply:
You won't be able to actually do this until after this is merged though since aab_info.rb isn't on the main branch yet :)